### PR TITLE
B #2363 fixed IPAM free addr doesn't work.

### PR DESCRIPTION
**Pull request by al3xhh**
Cherrypick 5.6

### CODE REVIEW

File: src/vnm/AddressRangePool.cc [hunk 1]
```diff
@@ -196,6 +195,8 @@ int AddressRangePool::rm_ar(unsigned int ar_id, string& error_msg)
 
     AddressRange * ar_ptr = it->second;
 
+    ar_ptr->free_addr(ar_id, error_msg);
```
Comment (rsmontero) [inline src/vnm/AddressRangePool.cc]: Why is this needed?  rm_ar only works on AR without addresses in use, so it would not be needed to free them.

Also free_addr uses index of an address (a IP for example)  not the ID of an address range.

Comment (al3xhh) [reply src/vnm/AddressRangePool.cc]: That is needed to call the IPAM driver and remove the AR using the driver. In my case I'm developing the Packet IPAM driver, so without that, the AR is not delete in Packet, it is just delete in OpenNebula.
